tests/attestation_bad_config.rs

```diff
@@ -0,0 +1,98 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: AGPL-3.0-only
+
+use steward::Config;
```

Comment:
These are isolated unit tests, which do not belong in `tests/integration`

Reply:
Resolved, files deleted